src/util.cr

```diff
@@ -33,5 +33,13 @@ module Wsman
         log.info("Not removing #{path}, does not exist.")
       end
     end
+
+    def self.set_owner(user, group, file, recursive = false)
+      Wsman::Util.cmd("chown", [recursive ? "-R" : "", "#{user}:#{group}", file])
+    end
+
+    def self.set_permission(perm, file, recursive = false)
+      Wsman::Util.cmd("chmod", [recursive ? "-R" : "", perm, file])
```

Comment:
Crystal stdlib provides an API for this. Please take a look [here](https://crystal-lang.org/api/0.28.0/File.html#chmod%28path%3APath%7CString%2Cpermissions%3AInt%7CPermissions%29-class-method).